src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -63,6 +63,22 @@ public ObservableCollection<DynamoDropDownItem> Items
             }
         }
 
+        public override NodeInputData InputData()
+        {
+
+            return new NodeInputData()
+            {
+                Id = this.GUID.ToString("N"),
+                Name = this.Name,
+
+                Type = NodeInputTypes.selectionInput,
+                Description = this.Description,
+                //TODO investigate if this node type also needs to serialize it's selected index.
+                Value = this.Items[this.selectedIndex].Name,
+                Choices = this.items.Select(x => x.Name).ToList(),
```

Comment:
If there's a better way to provide this data, such that a UI that doesn't have all the nodes of the graph, just the inputs, can provide this dropdown to the user, we should investigate this.  If it doesn't make sense for a UI without the nodes to provide these particular dropdowns, for example if the output of the node is not just a string, then it probably shouldn't be listed here.  I designed this particular input type for allowing the user to choose from simple lists of strings.